app/src/js/middleware/request.js

```diff
@@ -59,33 +59,45 @@ export const requestMiddleware = ({ dispatch, getState }) => (next) => (action)
 
     const { id, type } = requestAction;
 
+    const defaultRequestConfig = {
+      paramsSerializer (params) {
+        return qs.stringify(params, { arrayFormat: 'brackets' });
+      }
+    };
+
+    const requestConfig = {
+      ...defaultRequestConfig,
+      ...requestAction
+    };
+
     const inflightType = `${type}_INFLIGHT`;
     log((id ? `${inflightType}: ${id}` : inflightType));
     dispatch({ id, config: requestAction, type: inflightType });
 
     const start = new Date();
-    return requestPromise(requestAction)
+    return axios(requestConfig)
       .then((response) => {
-        const { body, statusCode } = response;
-        if (+statusCode >= 400) {
-          const error = new Error(getErrorMessage(response));
+        const { data } = response;
+        const duration = new Date() - start;
+        log((id ? `${type}: ${id}` : type), `${duration}ms`);
+        return next({ id, type, data, config: requestAction });
+      })
+      .catch((error) => {
+        if (error.response) {
+          const { data, status } = error.response;
           return handleError(
             {
               id,
               type,
-              error,
+              error: data,
               requestAction,
-              statusCode
+              statusCode: status
             },
             next
           );
         }
-
-        const duration = new Date() - start;
-        log((id ? `${type}: ${id}` : type), `${duration}ms`);
-        return next({ id, type, data: body, config: requestAction });
-      })
-      .catch((error) => handleError({ id, type, error, requestAction }, next));
+        handleError({ id, type, error, requestAction }, next);
```

Comment:
Does this still get called anymore?  Or is it just safety?  IDK.

Reply:
Yeah really just a safety. If the error is coming from some place other than the api response, the error won't have a response property, so we want to be able to handle that